cmd/atlas/internal/cmdapi/cmdapi.go

```diff
@@ -144,3 +116,8 @@ func parse(version string) (string, string) {
 	}
 	return version, u
 }
+
+// Version returns the current Atlas binary version.
+func Version() string {
+	return version
```

Comment:
did this in order to avoid breaking the binary build which expects version at the specific location